x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/evaluate_condition.ts

```diff
@@ -45,16 +45,14 @@ export const evaluateCondition = async ({
   compositeSize: number;
   filterQuery?: string;
   lookbackSize?: number;
-  startTime?: number;
+  startedAt: Date;
   logger: Logger;
 }): Promise<Record<string, ConditionResult>> => {
   const { metric, customMetric } = condition;
 
-  const to = startTime ? moment(startTime) : moment();
-
   const timerange = {
-    to: to.valueOf(),
-    from: calcualteFromBasedOnMetric(to, condition, nodeType, metric, customMetric),
+    to: startedAt.valueOf(),
+    from: calcualteFromBasedOnMetric(moment(startedAt), condition, nodeType, metric, customMetric),
```

Comment:
@simianhacker There's typo here. Change `calcualteFromBasedOnMetric` to `calculateFromBasedOnMetric`

Reply:
Fixed